packaging/tools/style-format.sh

```diff
@@ -70,6 +72,14 @@ for f in $*; do
         stylename="C"
     fi
 
+    if [[ $lang == c && $clang_version_warning == 0 ]]; then
+        # Check if clang-format is at the correct version, and if not, warn.
+        if ! clang-format --version | grep -o "version ${clang_format_version}\.[[:digit:]]\+\.[[:digit:]]\+" 1> /dev/null; then
```

Comment:
you can use `grep -q` instead of redirecting the output

Reply:
Using sed now